src/groundStation/groundStation.py

```diff
@@ -82,8 +84,26 @@ def __zmq__(self, addr):
 
     def __uart__(self, device):
         """ initialize uart interface """
-        libcsp.kiss_init(device, 115200, 512, 'uart')
+        ser = serial.Serial('/dev/ttyUSB0',                                      
```

Comment:
The path should be passed as a parameter. I believe the argparser has this parameter already